package.json

```diff
@@ -62,7 +63,8 @@
     "typescript": "^3.2.2",
     "vue": "^2.6.6",
     "vuepress": "^0.14.8",
-    "vuex": "^3.1.0"
+    "vuex": "^3.1.0",
+    "vuex-persist": "^2.0.0"
```

Comment:
The change is great, but I think tying this project to `vuex-persist` is an unnecessary coupling.

The change can very well live without any reference to `vuex-persist` and merely serve as an additional ability to set the `preserveState` option.

Reply:
First of all, it's development only. So will only be installed once you want to develop it. Since I wrote integration tests between the two libraries. How would I go ahead and test it without the library?

Reply:
@Kapulara I'm not sure, but you could install it as a peer dependency perhaps?